src/main/java/estaruma/cupcake/Toppings.java

```diff
@@ -0,0 +1,27 @@
+package estaruma.cupcake;
+
+
+// allow toppings
+public class Toppings extends BakeryItem {
```

Comment:
may be a separate class for each topping? Sugar, Chocolate, Nuts all extending CakeDecorator?